flink-core/src/main/java/org/apache/flink/api/java/typeutils/runtime/RowSerializer.java

```diff
@@ -60,7 +60,7 @@
 
 	public static final int ROW_KIND_OFFSET = 2;
 
-	private static final long serialVersionUID = 2L;
+	private static final long serialVersionUID = 1L; // legacy, don't touch
```

Comment:
nit: maybe add a comment that this can only be touched after support for 1.9 savepoints is ditched.